client/components/Reviews/ReviewListBase/useReviewListBase.js

```diff
@@ -9,10 +10,18 @@ export default function useReviewListBase(id, listReported, listUserReview, star
   const [isReviewsLoad, setIsReviewsLoad] = useState(false);
   const [listReviews, setListReviews] = useState([]);
 
-  useEffect(() => {
+  useEffect(async () => {
     setIsMoreReviews(true);
     setSort('relevant');
-    reviewsGET('reviews', id, 2, sort, setData, setIsReviewsLoad);
+    // reviewsGET('reviews', id, 2, sort, setData, setIsReviewsLoad);
+    const results = await moreReviewsGet({
+      string: 'reviews',
+      id,
+      count: 2,
+      sort,
+    });
+    setData(results);
+    setIsReviewsLoad(true);
   }, [id]);
 
   useEffect(() => {
```

Comment:
Great to see how you accomplished a couple of state changes inside useEffect method! I'm curious if there's a reason why const is capitalized here.